### PR TITLE
Add an end-to-end test

### PR DESCRIPTION
**Pull request by brandonchinn178**

### CODE REVIEW

**Comment by andywer** [comment]
Thanks for your efforts!

Do you think e2e tests make sense here? I am just wondering, since the user still writes most of the query themself, so we primarily test the user-supplied query.

I guess the main benefit is to check whether the query object that we create can actually be consumed by `pg`. The risk of having issues with that is low, though, as long as we unit test the returned query object's structure.

Happy to discuss.

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,11 @@ node_js:
   - 8
   - 10
 
+services:
```
Comment (andywer) [inline .travis.yml:7]: If we want to introduce e2e tests, we also need to provide a simple way to test locally, though.

Maybe something similar to how it's done in `pg-listen`?
https://github.com/andywer/pg-listen/blob/c34d0e1646d4b1f0f88cf021611e07cf05ad4ee5/package.json#L13

Comment (brandonchinn178) [reply .travis.yml:7]: Sure!

**Comment by brandonchinn178** [comment]
> Do you think e2e tests make sense here? I am just wondering, since the user still writes most of the query themself, so we primarily test the user-supplied query.

Yeah, basically what you said. I just want to make sure that what we send back can be consumed by pg. I think the unit tests are great and cover all the edge cases, but on the off chance that the API is all self-consistent, but it just doesn't integrate with pg, we want to catch that.

I have one other refactoring PR on the way (in order to fix #27) and I wanted to be extra sure I didn't break anything user side.

**Comment by andywer** [comment]
Nice!
